src/hatti/map/viewby.cljs

```diff
@@ -82,27 +46,29 @@
    eg. a bin for numbers/dates, an option for multiple/single selects."
   [field raw-answers ids]
   (let [fname (:full-name field)
-        ans-s (preprocess-answers field raw-answers)
-        answer->count (frequencies (flatten ans-s))
-        sorted (cond
-                 (or (f/categorical? field)
-                     (f/text? field)
-                     (f/calculate? field))
+        preprocessed-answers (preprocess-answers field raw-answers)
+        answer->count (frequencies (flatten preprocessed-answers))
+        sorted-answers (cond
+                 (or (form-utils/categorical? field)
```

Comment:
Yeah, that would be great, I haven't come across a clj* style checker 
that does this, but I'll take another pass googling

Okal Otieno wrote:
>
> In src/hatti/map/viewby.cljs 
> <https://github.com/onaio/hatti/pull/96#discussion_r47080370>:
>
> > @@ -82,27 +46,29 @@
> >     eg. a bin for numbers/dates, an option for multiple/single selects."
> >    [field raw-answers ids]
> >    (let [fname (:full-name field)
> > -        ans-s (preprocess-answers field raw-answers)
> > -        answer->count (frequencies (flatten ans-s))
> > -        sorted (cond
> > -                 (or (f/categorical? field)
> > -                     (f/text? field)
> > -                     (f/calculate? field))
> > +        preprocessed-answers (preprocess-answers field raw-answers)
> > +        answer->count (frequencies (flatten preprocessed-answers))
> > +        sorted-answers (cond
> > +                 (or (form-utils/categorical? field)
>
> I wonder if we can we get the style check to catch these?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/onaio/hatti/pull/96/files#r47080370>.
>